### PR TITLE
Milestone: 7 Create accessibility

### PR DESCRIPTION
**Pull request by Phone-Khaing-Hein**
Page titles - no issues found
Image text alternatives** - Issues found at
https://github.com/Rimuru-Tempest-26/Portfolio/blob/85de1e29b30ce8f5e8ed8fdbe49b62051e53fa6d/index.html#L92-L94
Text headings - no issues found
Color contrast - no issues found
Resize - no issues found
Interaction - no issues found
Moving content - no issues found
Multimedia - no issues found
The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,9 +89,9 @@ <h2 class="card-title">Tonic</h2>
                 <h2 class="card-title">Multi-Post Stories</h2>
                 <ul class="position">
                     <li class="name">FACEBOOK</li>
-                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li><img src="images/icons/bullet.svg" alt="bullet icon"></li>
                     <li class="status">Full Stack Dev</li>
-                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li><img src="images/icons/bullet.svg" alt="bullet icon"></li>
```
Comment (iambenkis) [inline index.html:115]: - [ ] Keep doing the great work :+1:

File: index.html [hunk 2]
```diff
@@ -67,9 +67,9 @@ <h3 class="connect-link">
                 <h2 class="card-title">Tonic</h2>
                 <ul class="position">
```
Comment (iambenkis) [inline index.html:89]: - [x] Keep doing the great work with the heading hierarchies :+1:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey Team (@Rimuru-Tempest-26 , @helina42 ) :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good
:heavy_check_mark: Can be maximized upto 200%

Good job on implementing some of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

- [ ]  Your logo does not have  enough contrast of 3.9:9. Please add enough contrast for this button , the color of when the hovering , focusing on the element should be contrasted :+1: 

            <a href="index.html" class="brand">My Logo</a>

Screenshot from 2023-02-28 16-42-36



# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Phone-Khaing-Hein) [reply index.html:115]: Thanks.

File: index.html (hunk 2 above)
Comment (Phone-Khaing-Hein) [reply index.html:89]: Thanks.

**Pull request closed without merging**